pennylane/templates/subroutines/select.py

```diff
@@ -358,19 +358,23 @@ def resource_params(self):
         }
 
     def _flatten(self):
-        return (self.ops), (
+        return tuple(), (
+            self.ops,
             self.control,
             self.hyperparameters["work_wires"],
             self.partial,
         )
 
+    # pylint: disable=arguments-differ
     @classmethod
-    def _primitive_bind_call(cls, *args, **kwargs):
-        return cls._primitive.bind(*args, **kwargs)
+    def _primitive_bind_call(cls, ops, control, **kwargs):
+        return super()._primitive_bind_call(ops=ops, wires=control, **kwargs)
```

Comment:
Again, `ops` is a list of operators, and each operator needs to be treated as a tracer.